<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
  	<node name="lane_detection" pkg="autovalet_lane_detection" type="autovalet_lane_detector.py" output="screen"/>

</launch>
